### DISCUSSION TITLE
classes/class_node2d

### DISCUSSION THREAD

**Discussion by giscus** [godotengine/godot-docs-user-notes#110] [User-contributed notes]
# classes/class_node2d

Inherits: CanvasItem< Node< Object Inherited By: AnimatedSprite2D, AudioListener2D, AudioStreamPlayer2D, BackBufferCopy, Bone2D, Camera2D, CanvasGroup, CanvasModulate, CollisionObject2D, CollisionP...

https://docs.godotengine.org/en/stable/classes/class_node2d.html

**Comment by HephepTeam** [comment]
The units of skew and global_skew properties are not explained here, it should be added. And actually, in Godot 4.2.2 at least, skew is expressed in radians when used in script, but in degrees in the inspector. It should be clarified.

**Reply by Calinou** [reply]
- Done: https://github.com/godotengine/godot/pull/96583

**Comment by nathancarter** [comment]
Short description at top of page: "A 2D game object, inherited by all 2D-related nodes. Has a position, rotation, scale, and Z index."

But it does not appear to have a Z index.  Not only would it be good to remove that, but it would be nice to explain what has replaced it, since it's a common term one searches for in the docs to know how to deal with layering order.

**Reply by Calinou** [reply]
Pull request opened:

- https://github.com/godotengine/godot/pull/99238

**Comment by follyfoxe** [comment]
For some reason get_global_transform_interpolated() is nowhere to be seen for 2d nodes (yet Node3D does seem to have a similar method)
See https://github.com/godotengine/godot-proposals/discussions/5786
If you urgently need a solution I posted a workaround on the comments there
